src/piadm.sh

```diff
@@ -195,6 +196,104 @@ DEFAULT_URL_PREFIX=https://us-central.manta.mnx.io/Joyent_Dev/public/SmartOS/
 # Default path for piadm's configuration
 PIADM_CONF=/var/piadm/piadm.conf
 
+# fetch_checksum
+#
+# Fetches a MD5 hash using a platform file as the key.
+#
+# Arguments:
+#   $1 - URL from where to fetch the PI from. The name of the PI
+#   	 is expected to be the last path component of the URL,
+#   	 or the literal string "latest".
+#
+# Environment Variables:
+#   PIADM_NO_CHECKSUM     - If set to 1, skips checksum validation.
+#   PIADM_CHECKSUM_URL    - If set, overrides the default md5sums.txt URL
```

Comment:
Same advice per the man page about `md5sums.txt`.  It's the current default value, and the only one available.